lib/http.js

```diff
@@ -121,17 +121,17 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     validate({query: 'bedrock-profile-http.account'}),
     asyncHandler(async (req, res) => {
       const {account: {id: accountId}} = (req.user || {});
-      const {account} = req.query;
+      const {profileAgentId} = req.params;
+      // this throws NotFoundError
+      const profileAgentRecord = await profileAgents.get({id: profileAgentId});
       // FIXME: Use http-sigs and zcaps for
       // TODO: Add permissions to allow access for admin
-      if(account !== accountId) {
+      if(profileAgentRecord.profileAgent.account !== accountId) {
```

Comment:
I think this would be a bit safer to handle cases where no `account` is set (for integrations):

```suggestion
      if(!(profileAgentRecord.profileAgent.account &&
        profileAgentRecord.profileAgent.account === accountId)) {
```